modin/pandas/io.py

```diff
@@ -120,7 +122,7 @@ def read_csv(
     memory_map=False,
     float_precision=None,
     storage_options: StorageOptions = None,
-):
+):  # noqa: D103
```

Comment:
Why is `noqa: D103` used?

Reply:
This allows us to disable the check that the function does not have a docstring.

Reply:
Okay, I see.